<launch>
    <arg name="is_sync" default="true"/> 
    <arg name="rate" default="20"/>
    <arg name="is_print" default="false"/>

    <arg name="config_path" default="/home/dji/SwarmConfig/camera_config.yaml" />

    <node name="stereo" pkg="ptgrey_reader" type="sync_multi_reader" output="screen">
        <param name="is_pub" type="bool" value="true"/>
        <param name="is_show" type="bool" value="false"/>
        <param name="is_print" type="bool" value="true"/>
        <rosparam>
            # common setting and default individual settings
            cam_cnt: 2
            frameRate:  20
            gain: 10.0
            is_auto_shutter: false
            is_sync: true
            shutter: 5.0
            brightness: 10.1
            exposure: 0.4

            # individual setting for camera 0
            camera0:
                serialNum: 17302082
                topic:     /stereo/left/image_raw
                topic_roi: /stereo/left/image
                is_roi:    true
                size_x:    1280
                size_y:    1024
                center_x:  640
                center_y:  512
                cropper_x: 1280
                cropper_y: 1024
                down_sample_scale: 0.5
            camera1:
                serialNum: 17533322
                topic:     /stereo/right/image_raw
                topic_roi: /stereo/right/image
                is_roi:    true
                size_x:    1280
                size_y:    1024
                center_x:  640
                center_y:  512
                cropper_x: 1280
                cropper_y: 1024
                down_sample_scale: 0.5

        </rosparam>
    </node>
</launch>